frontend/src/Providers.tsx

```diff
@@ -6,6 +6,13 @@ import { ReactNode } from 'react';
 import { AuthProvider } from 'react-oidc-context';
 import { BrowserRouter } from 'react-router-dom';
 
+declare module '@tanstack/query-core' {
+  interface QueryMeta {
+    autoClose?: false | number;
```

Comment:
why is it not boolean or number?

Reply:
its the same type as defined in toast options:

/**
     * Set the delay in ms to close the toast automatically.
     * Use `false` to prevent the toast from closing.
     * `Default: 5000`
     */
    autoClose?: number | false;
 **/